### PR TITLE
Support LocalClient.{delete,move} operations (expand testing too)

### PR DESCRIPTION
**Pull request by matt4d617474**
Note also that in order to more fully test, I made some other minor changes.

### CODE REVIEW

**Comment by matt4d617474** [comment]
Hmmmm... not sure why coveralls is thinking coverage percentage decreased.  I believe I added tests for everything I wrote and even tested beyond that to have confidence in the integration test approach with testfixtures.TempDirectory.

**Comment by matt4d617474** [comment]
Any hope of getting this reviewed soon?  Or should I focus my efforts elsewhere.  Thanks!

**Comment by matt4d617474** [comment]
Thanks for reviewing!  I'm happy to make the suggested changes in the next couple days as my schedule permits.  The one thing I'm a bit hazy on, due to my GitHub ignorance... if I were to squash the commits I basically need to reset the branch.  I'm used to squashing pre-push but haven't squashed post-push ever.  I hesitate to change my quasi-published history here (e.g., would the diffs and review still make sense?).  Would it be preferable to instead submit a separate pull request where everything is squashed?

Thanks!

**Comment by dsoprea** [comment]
The PR will always represent the difference between my project/branch and yours. Whether you have two commits one day, five commits the next, or one the day after, your PR will always represent the whole, big-picture difference. Since you're only allowed to have one PR open at a time, there's no ambiguity. Most review systems work the same way.

Squashing contributions is generally considered good practice, anywhere. It's always going to be easier to wrap your head around third-party contributions when they're neatly packaged into single commits. It's also obviously perfectly acceptable to also keep whatever you have (if that helps you) and to just squash right before we're ready to merge, too.

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@ venv*
 .tox
 /pip-selfcheck.json
 /man
+*~
```
[the remark this answers is not in the document]
Comment (jfthuong) [reply .gitignore:18]: @dsoprea  How to drop a specific change from a PR?

File: svn/local.py [hunk 2]
```diff
@@ -37,6 +37,12 @@ def add(self, rel_path):
             [rel_path],
             wd=self.path)
 
+    def delete(self, rel_path):
```
[the remark this answers is not in the document]
Comment (jfthuong) [reply svn/local.py:40]: This functionality is now in master under the name `remove`.

**Comment by dsoprea** [comment]
Closed due to ignored conflicts.

**Pull request closed without merging**